doc/optimizations.md

```diff
@@ -102,6 +109,22 @@ If you have followed this guide, your geoserver probably also uses our recommend
 If not, you should create a custom ```controlflow.properties``` file in your geoserver "data dir".  
 Please refer to the [control-flow module documentation](http://docs.geoserver.org/latest/en/user/extensions/controlflow/index.html) for the syntax.
 
+### More fonts
```

Comment:
We should probably remove the same wording from https://github.com/georchestra/georchestra/blob/15.06/doc/post-deploy_config.md#fonts 

And clearly say that this should be performed on the machine hosting the geoserver.